### DISCUSSION TITLE
I'm at step Testing 1 in the build phase but no LEDs

### DISCUSSION THREAD

**Discussion by gurlen** [thehighnibble/thehighnibble.github.io#52] [IMSAI8080esp replica]
I'm at the point in assembly where all ICs are in place, I have tested that all LEDs worked in a previous step, but after I put the IMSAI8080esp into run, no LEDs are illuminated.  Wifi is up, and I can use the web interface, but no LEDs.  I verified that all ICs are in the proper orientation, and looked for bad solder joints.  With no schematic, does anyone know what would cause all the LEDs to not be triggered?

**Comment by gurlen** [comment]
I also verified that pins 10-14 of the 74hc595n's all have continuity between each other and to the esp32

**Comment by GrantMeStrength** [comment]
I think this is expected at this stage.. I mean, do you have it actually up and running?

**Comment by gurlen** [comment]
Nevermind - I did not know that I had to connect and leave connected S1 common to top - I thought momentary was enough... hopefully that is my only dumb move of the day!
